### ISSUE TITLE
Can we consider supporting MAC M1?

### ISSUE THREAD

**Issue by hibisicus** [RamonUnch/AltSnap#440]
My compyter is MAC M1pro, i hope this programe can supper my computer

**Comment by RamonUnch** [comment]
This program is very much Windows specific so no port to MacOS is planned.
If
 you are using Windows for ARM on your MAC, then it should be possible to build AltSnap for ARM, unfortunately gcc only supports x86 Windows cross compiling. It should be possible using a recent MSVC to generate an ARM64/ARMv7 build, but I have not yet looked into it, and I would be unable to test the binaries.

I ultimately would like to support all CPU Windows supports, ARM/PowerPC/MIPS/Alpha...

**Comment by hibisicus** [comment]
fine,thank you~

**Comment by RamonUnch** [comment]
Yep it did not live for long, I think you only got Windows NT3.x and NT4 that supported Alpha AXP IIRC.
Other than x86 and ARM, supporting other CPUs would be just be for fun on retro hardware. I mean even PowerPC and MIPS Windows was only supported up to NT4. Also Itanium is mostly dead, even though much more recent Windows versions exist for it.

**Comment by LoganDark** [comment]
> Yep it did not live for long, I think you only got Windows NT3.x and NT4 that supported Alpha AXP IIRC. Other than x86 and ARM, supporting other CPUs would be just be for fun on retro hardware. I mean even PowerPC and MIPS Windows was only supported up to NT4. Also Itanium is mostly dead, even though much more recent Windows versions exist for it.

Does AltSnap still support really old Windows versions?

**Comment by LoganDark** [comment]
NT4 supports Alpha, so that checks out :)

**Comment by NebulusIO** [comment]
> My compyter is MAC M1pro, i hope this programe can support my computer

Would be great to get similar functionality although best I could find is using BTT (Better Touch Tool)
